backend/db/utils.py

```diff
@@ -48,30 +50,35 @@ def refresh_termhub_core_cset_derived_tables(con: Connection, schema: str):
         'cset_members_items_plus',
         'codeset_counts',
         'all_csets',
-        # 'csets_to_ignore',            not using for now
+        'csets_to_ignore',
     ]
     views = [
         'cset_members_items_plus',
-        # 'csets_to_ignore',            not using for now
+        'csets_to_ignore',
     ]
+
     # Create new tables and backup old ones
+    print('Derived tables')
     t0 = datetime.now()
+    hash_num = '_' + str(randint(10000000, 99999999))
```

Comment:
Changed method of re-creating indexes. Now adds a random hash suffix rather than recreating at the end.